src/pkg/repository/repository.go

```diff
@@ -367,10 +366,7 @@ func (r repository) BackupDetails(
 	if b.Version >= version.OneDrive1DataAndMetaFiles && b.Version < version.OneDrive3IsMetaMarker {
```

Comment:
If we do bump up the version, we can use this to limit this loop to only ones under the version.